plugins/window_size/macos/Classes/FLEWindowSizePlugin.m

```diff
@@ -160,6 +162,13 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
     methodResult =  @[
                               @(ChannelRepresentationForMaxDimension(size.width)),
                               @(ChannelRepresentationForMaxDimension(size.height)) ];
+  } else if ([call.method isEqualToString:kSetWindowTitleMethod]) {
+    NSString *title = call.arguments;
+    self.flutterView.window.title = title;
+    methodResult = nil;
+  } else if ([call.method isEqualToString:kSetWindowTitleRepresentedUrlMethod]) {
+    NSURL *representedURL = [NSURL URLWithString:call.arguments];
+    self.flutterView.window.representedURL = representedURL;
```

Comment:
Add a `methodResult = nil` here for consistency with the existing setters.

Reply:
Done